test/functional/test-viewport.js

```diff
@@ -191,27 +191,27 @@ describe('Viewport', () => {
     // Shouldn't call updatePaddingTop since it hasn't changed.
     let bindingMock = sandbox.mock(binding);
     bindingMock.expects('updatePaddingTop').never();
-    viewerViewportHandler();
+    viewerViewportHandler({paddingTop: 19});
     bindingMock.verify();
 
     // Should call updatePaddingTop.
     bindingMock = sandbox.mock(binding);
     viewport.fixedLayer_.updatePaddingTop = () => {};
-    viewerMock.expects('getPaddingTop').returns(21).atLeast(1);
-    bindingMock.expects('updatePaddingTop').withArgs(21).once();
-    viewerViewportHandler();
+    viewerMock.expects('getPaddingTop').returns(0).atLeast(1);
```

Comment:
Tests complain with:

> Viewport "after each" hook for "should update padding when changed only" FAILED
> Expected getPaddingTop([...]) at least once

I think b/c we are now reading `paddingTop` from the event and not the viewer, some minor adjustments here might be necessary. PTAL.